### DISCUSSION TITLE
Event Templates

### DISCUSSION THREAD

**Discussion by prumand** [pretix/pretix#4421] [Q&A]
I'm running an evaluation and migration from a self-coded ticket system to pretix (Hosted version) for one of my customers. While trying things out I stumbled feature, which I couldn't setup in pretix:
They have three roles, admin, coordination, event-organization.
1. Admins are the technical owners of the application.
2. Coordinators are in charge of inviting and managing event-organizers and creating templates from which event organizers can choose.
3. Event Organizers just fill in the dates and create events from templates. Later the manage the participants as well (not billing).

Is there any way to create "templates" for events and let different user create them? I would also be happy with known solution to similar problems, even if they are slightly different. A experimenten with the "event metadata", but I didn't quite cut it.

**Comment by raphaelm** [comment]
There's no specific template, but events can be cloned and you can give different people access to different events to create as templates. I'm not sure from the top of my head  how much you can restrict the level of access they have on the cloned events, I believe you need pretty full access to an event to clone it.

**Reply by prumand** [reply]
Thank you very much. I thought about something like this. That seems pretty close. Maybe I need to build a small app to retain their old flow.
